.clang-format

```diff
@@ -0,0 +1,3 @@
+BasedOnStyle: Mozilla
```

Comment:
there's something I strongly dislike in the default mozilla style that sometimes puts even the function body on a single line, please consider disabling that. It's probably `AllowShortFunctionsOnASingleLine` but there may be other related settings that influence it, I don't remember right now.

Reply:
I tamed this long ago: 
https://github.com/usnistgov/ndn-dpdk/blob/f02f2249a302ea9d3b8ab91ddefa3c58b104633a/.clang-format#L3
(only compatible with clang-format-15)

Reply:
what are you trying to point to? isn't that the setting I already mentioned?

Reply:
I gave the value (*Empty*) of this parameter.

Reply:
I used `None` in the WIP config on gerrit. I dislike `Empty` too, although it's not as horrible as the mozilla default.

Reply:
Done - I've no preference here.